_posts/2020/2020-04-23-instagram-blog-post.md

```diff
@@ -0,0 +1,83 @@
+
+## Create an agenda for events and fairs
+
+Throughout the year ePages is organizing internal events like our [hackathon](/blog/events/this-years-hackathon-a-travel-through-time/){:target="_blank"}, our Sales Kick-off event, or the [ePages YOU](/blog/events/epages-you-2019-from-a-perspective-of-two-newbies/){:target="_blank"}.
+But ePages is also participating in external events such as conferences and fairs.
+These events can also provide perfect content for your company Instagram channel.
+In order to keep them in mind create an agenda where you list all internal and external events.
+Note also which team or employees do visit the external events.
+Thus, you can remind them on time to take some nice pictures of the event.
+
+## Create a content plan
+
+In order to coordinate your content on Instagram you can create a content plan.
+This is a calender where you can note which picture do you want to share when.
+That makes it easier for you to plan your content sharing in advance.
+
+We created a content plan at the start of our Instagram channel.
+But it didn't work for us, yet.
+We're are still a little channel and at the moment we're very flexible with sharing our content.
+So, we're deciding each week what content we would like to share on Instagram.
+But we keep the content plan in mind - it might support us in the future.
+
+## Consistent visual presentation of your Instagram profile
+
+Create a profile picture which fits to the corporate design of your company.
+This will establish a stronger connection to the company.
+If you have a highlights section, create cover images that matches to the companys corporate design as well.
+These points will support a consistent visual presentation of your Instagram profile.
```

Comment:
What do you think about "style"? "Visual presentation" felt so abstract...

This will support a consistent style of your Instagram profile.
And of course, also keep in mind to have a matching style of the images shared in your Instagram feed.